### ISSUE TITLE
Support for JIRA links (blocks, is blocked by, etc)

### ISSUE THREAD

**Issue by asg0451** [ahungry/org-jira#38]
It would be nice to have JIRA links in the org output of this package, perhaps implemented with org internal links. Any thoughts?

**Comment by ahungry** [comment]
So, when a Jira entry has a link in it's custom markdown, you would like it to be an org style link?

**Comment by asg0451** [comment]
no, the issue links, as in (jira filter query) issue in linkedIssues(ISSUENAME, 'blocks')

i played around with libjira internals and issue links seems to be an included field. should be possible to do this. i may take a crack at it myself. do you take PRs?

**Comment by ahungry** [comment]
Absolutely!
